src/codegen/function_body_chunk.rs

```diff
@@ -367,6 +346,17 @@ impl Builder {
         }
     }
 
+    fn is_uninitialized_var(
+        &self,
+        name: &String,
```

Comment:
```suggestion
        name: &str,
```

I'm surprised that clippy didn't complain about this one